daal4py_bench/requirements.txt

```diff
@@ -3,3 +3,4 @@ pandas < 1.3.0
 daal4py
 openpyxl
 tqdm
+requests
```

Comment:
Might be missing some context here - why we need requests module? Per my understanding we do not suppose to use it?

Reply:
This is to replace the urllib.request.urlretrieve used in load_data. The urllib library failed a check in code factor so I used requests instead.